src/plugins/github/cacheId.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import {type RepoId, repoIdToString} from "./repoId";
+
+/**
+ * A derived ID to reference a cache layer.
+ * Instances of this type will be:
+ * - Deterministic
+ * - Unique for this plugin
```

Comment:
Unique per RepoId

Reply:
Currently interchangeable. Though should we ever decide to add a second type to derive from, they mustn't conflict. Subtle implementation note for the future :]